packages/app/src/runner/SpecRunnerRunMode.vue

```diff
@@ -57,7 +58,8 @@
           <div
             v-show="!autStore.scriptError"
             :id="RUNNER_ID"
-            class="origin-top-left viewport"
+            class="viewport"
+            :class="screenshotStore.isScreenshotting ? 'origin-top-left' : 'origin-top'"
```

Comment:
How come this is slightly different to https://github.com/cypress-io/cypress/pull/21019/files#diff-664b062627eb76f574368ed4f4a7d4e76a18393da8663e7ff64b46fe6020a433L78 ?

Reply:
Great catch, I will double check open mode screenshots, it seems like they would probably have the same problem this is solving, so these should match.

Reply:
Turned out both are working fine and this conditional setting of origin was able to be removed.